### PR TITLE
[Feature] add mongodb id encryption feature 

### PR DESCRIPTION
**Pull request by BlueWhaleKo**
TL; DR;
add mongodb id encryption feature

### CODE REVIEW

File: utils/mongodb/encrypt.test.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      },
+      {
+        Id: '4qxrK3r1kNCyB2j82P6V',
+        username: 'bar',
+      },
+    ];
+
+    expect(encodeDocument(user)).toStrictEqual(expected);
+  });
+});
```
Comment (Jeonghyun-Kim) [inline utils/mongodb/encrypt.test.ts:64]: Please add some test cases for ending with '_ids'

File: utils/mongodb/encrypt.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    });
+    return Object.fromEntries(clone);
+  }
+
+  return input;
+}
+
+export function encodeDocuments<U>(input: Array<any>): Array<U> {
+  return input.map((v) => encodeDocument<U>(v));
+}
```
Comment (Jeonghyun-Kim) [inline utils/mongodb/encrypt.ts:47]: this can be covered by `encodeDocument()`

**Review by Jeonghyun-Kim — CHANGES_REQUESTED** [review]

**Comment by Jeonghyun-Kim** [comment]
동형님 최고 👍

File: utils/mongodb/encrypt.test.ts (hunk 1 above)
Comment (BlueWhaleKo) [reply utils/mongodb/encrypt.test.ts:78]: Resolved by https://github.com/coxwave/nextjs-template/pull/1/commits/94469cff027e34bb23183be5008f92f56e6902d1

File: utils/mongodb/encrypt.ts (hunk 2 above)
Comment (BlueWhaleKo) [reply utils/mongodb/encrypt.ts]: Resolved by https://github.com/coxwave/nextjs-template/pull/1/commits/b84c1910cf05e0565bd7b3725d0e3e2fc4974460

**Review by Jeonghyun-Kim — APPROVED** [review]
💕

**Pull request merged**